src/identities.js

```diff
@@ -68,7 +69,13 @@ class Identities {
   }
 
   static async createIdentity (options = {}) {
-    const keystore = options.keystore || Keystore.create(options.identityKeysPath || identityKeysPath)
+    if(!options.keystore) {
```

Comment:
Is it not possible to create a keystore here if none is passed?

Reply:
Fixed!